src/webviews/apps/home/home.ts

```diff
@@ -17,12 +18,31 @@ import '../shared/components/button';
 import '../shared/components/code-icon';
 import '../shared/components/feature-badge';
 import '../shared/components/overlays/tooltip';
+import '../shared/components/onboarding/progress-line';
+import '../shared/components/onboarding/progress-tracker';
+import '../shared/components/onboarding/onboarding-item';
+import '../shared/components/onboarding/onboarding-item-group';
+import '../shared/components/onboarding/onboarding-checkbox';
```

Comment:
We should import components in the files they are being used in, so most of these import should move into the main web component.

Reply:
done